### PR TITLE
Hydroponics fixes

### PR DESCRIPTION
**Pull request by CoZarctan**
Fixes issues #349 #415 #303 

-Duplicates hydroponics tray fix from Baystation issue 11661
-makes splattering fruit use the reagent splash() proc rather than adding its contents directing into mobs' blood
-Adds rolling papers from /tg/ for use with the drying rack. I think that ambrosia produces less space drugs than they used to since the plant update, so I had to raise the chem ingestion multiplier for cigarettes so that you could actually get high from rollies. This may have to be tweaked further.

### CODE REVIEW

**Comment by scrdest** [comment]
Screw making Botany usable, #303 is fixed by this PR. :+1: :heavy_multiplication_x: :100: 

Seriously though, it does look alright code-wise at a glance. Tested?

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 1]
```diff
@@ -117,7 +117,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 					if(H.species.flags & IS_SYNTHETIC)
 						return
 
-				reagents.trans_to_mob(C, REM, CHEM_INGEST, 0.2) // Most of it is not inhaled... balance reasons.
+				reagents.trans_to_mob(C, REM, CHEM_INGEST, 1) // Most of it is not inhaled... balance reasons.
```
Comment (Glloyd) [inline code/game/objects/items/weapons/cigs_lighters.dm]: this is the only part that i'm worried about, but we'll see how it plays out. maybe .5 will be better, but i'm not sure how much ambrosia was nerfed, so as with most balancing things this'll have to be tweaked.

**Comment by Glloyd** [comment]
Besides that :+1:

**Comment by CoZarctan** [comment]
Hard to test every possible edge-case, but things like throwing thermite tomatoes at people and then setting them on fire works. 
I added a pack of 10 rolling papers to botany, but after those run out the only way to replenish them is by praying to Stoner Jesus. How do y'all feel about a cargo crate or something?

**Comment by Glloyd** [comment]
Yeah. If there's a cargo crate with hydro supplies, toss them in there.

**Comment by scrdest** [comment]
Why not add them to autolathe as a wood product?

**Comment by CoZarctan** [comment]
Well, IRL I believe rolling paper is not made from wood, but instead other plant fibers. (Should we care about that?) I could see it potentially as a bio-generator item. This is why I asked, there are a few places they could conceivably go.

**Comment by Glloyd** [comment]
If you're going to add it to something, add it to the pulp and paper processor

**Comment by liction15** [comment]
rip deadly ggnore fruit
